packages/devtools_app/lib/src/utils.dart

```diff
@@ -71,6 +71,9 @@ String prettyPrintBytes(
   int mbFractionDigits = 1,
   bool includeUnit = false,
 }) {
+  if (bytes == null) {
+    return null;
+  }
```

Comment:
should we assert that bytes != null here instead? or perhaps just return the empty string? why is this method called on a null value?

Reply:
An example is when we first create the VM statistics view and we build the page before we've gotten any data from the service. Returning null here saves us from having to do null checks in multiple places (at least 4 instances in `vm_statistics_view.dart`). I'm happy to change the approach if you have any suggestions.